File: src/main/java/dev/slne/surf/lobby/jar/command/subcommand/setting/ParkourSettingCommand.java
package dev.slne.surf.lobby.jar.command.subcommand.setting;

import dev.jorel.commandapi.CommandAPICommand;

public class ParkourSettingCommand extends CommandAPICommand {

  public ParkourSettingCommand(String commandName) {
    super(commandName);

    withPermission("jumpandrun.command.setting");

    withSubcommand(new ParkourSettingAreaCommand("setArea"));
    withSubcommand(new ParkourSettingSpawnCommand("setSpawn"));
    withSubcommand(new ParkourSettingStartCommand("setStart"));
    withSubcommand(new ParkourSettingMaterialListCommand("listMaterials"));
    withSubcommand(new ParkourSettingMaterialRemoveCommand("removeMaterial"));
    withSubcommand(new ParkourSettingMaterialAddCommand("addMaterial"));
    withSubcommand(new ParkourSettingSaveCommand("saveConfiguration"));
  }
}

File: src/main/java/dev/slne/surf/lobby/jar/command/subcommand/setting/ParkourSettingCommand.kt
package dev.slne.surf.lobby.jar.command.subcommand.setting

import dev.jorel.commandapi.CommandAPICommand

class ParkourSettingCommand(commandName: String) : CommandAPICommand(commandName) {
    init {
        withPermission("jumpandrun.command.setting")

        withSubcommand(ParkourSettingAreaCommand("setArea"))
        withSubcommand(ParkourSettingSpawnCommand("setSpawn"))
        withSubcommand(ParkourSettingStartCommand("setStart"))
        withSubcommand(ParkourSettingMaterialListCommand("listMaterials"))
        withSubcommand(ParkourSettingMaterialRemoveCommand("removeMaterial"))
        withSubcommand(ParkourSettingMaterialAddCommand("addMaterial"))
        withSubcommand(ParkourSettingSaveCommand("saveConfiguration"))
    }
}
